pytorch_blade/bazel_build.py

```diff
@@ -114,14 +114,8 @@ def run(self, extdir=None, srcdir=None, build_temp=None):
             bazel_cmd, shell=True, env=env, executable="/bin/bash"
         )
 
-        ext_so_fpath = "src/_torch_blade.so"
-        ral_so_fpath = "external/org_tensorflow/tensorflow/compiler/mlir/xla/ral/libral_base_context.so"
-        disc_bin_fpath = (
-            "external/org_tensorflow/tensorflow/compiler/mlir/disc/disc_compiler_main"
-        )
-
-        for fpath in [ext_so_fpath, ral_so_fpath, disc_bin_fpath]:
-            fpath = os.path.realpath(os.path.join(bazel_bin_dir, fpath))
+        for fpath in open("bazel_target_cfg.txt"):
```

Comment:
Do you want users to extend this configuration file? If not, maybe some const string instead of an additional text file is enough. 
If we want users to do something on the configuration file, we also need a guide doc to tell them how to do that.

Reply:
As comments said, I added some comments here. It does want users to extend it, but those users are mainly ourselves. So I don't want to add it to documentation since it might not be ready for users.